### PR TITLE
Remove Jackson usage in Attestation, Models Repository, Quantum Jobs, and Video Analyzer Edge

### PR DESCRIPTION
**Pull request by alzimmermsft**
# Description

Removes the usage of Jackson in the following packages:

- azure-iot-modelsrepository
- azure-media-videoanalyzer-edge
- azure-quantum-jobs
- azure-security-attestation

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/videoanalyzer/azure-media-videoanalyzer-edge/swagger/src/main/java/VideoAnalyzerEdgeCustomization.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+import com.github.javaparser.javadoc.Javadoc;
+import com.github.javaparser.javadoc.description.JavadocDescription;
+import org.slf4j.Logger;
+
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ * Customization class for Blob Storage.
```
Comment (srnagar) [inline sdk/videoanalyzer/azure-media-videoanalyzer-edge/swagger/src/main/java/VideoAnalyzerEdgeCustomization.java:23]: nit: Fix javadoc.

File: sdk/attestation/azure-security-attestation/pom.xml [hunk 2]
```diff
@@ -37,7 +37,6 @@
     <javaModulesSurefireArgLine>
```
Comment (srnagar) [inline sdk/attestation/azure-security-attestation/pom.xml:37]: Add azure-json dependency to all 3 libraries in this PR.

**Review by srnagar — APPROVED** [review]

**Comment by alzimmermsft** [comment]
@microsoft-github-policy-service rerun

**Comment by alzimmermsft** [comment]
@microsoft-github-policy-service rerun

**Pull request merged**